### ISSUE TITLE
打开没代码啊

### ISSUE THREAD

**Issue by 1286667673** [Neerajj9/CSRNet-keras#17]

**Comment by aditya-malte** [comment]
Hello,
A further description would prove very helpful in understanding the problem

Thanks,
Aditya

**Comment by 1286667673** [comment]
what's the problem
70/700 [==>...........................] - ETA: 1:28:58 - loss: nan - mean_squared_error: nan['data\\part_A_final/train_data\\images\\IMG_137.jpg']
(73, 128, 1)
 71/700 [==>...........................] - ETA: 1:28:09 - loss: nan - mean_squared_error: nan['data\\part_A_final/train_data\\images\\IMG_139.jpg']
(46, 68, 1)
 72/700 [==>...........................] - ETA: 1:27:07 - loss: nan - mean_squared_error: nan['data\\part_A_final/train_data\\images\\IMG_181.jpg']
loss is nan

**Comment by aditya-malte** [comment]
Hello,
Please check if you have performed the various steps such as preprocessing, loading the pretrained model correctly, as we are not facing any such issues during training.
Thank you

**Comment by Somyarani1113** [comment]
> Hello,
> Please check if you have performed the various steps such as preprocessing, loading the pretrained VGG model correctly, as we are not facing any such issues during training.
> Thank you

Hi @aditya-malte, 
   I have cloned the repo and followed all the steps in preprcoessing.ipnyb and model.ipynb but still got the "loss:nan".

**Comment by YashK07** [comment]
> > Hello,
> > Please check if you have performed the various steps such as preprocessing, loading the pretrained VGG model correctly, as we are not facing any such issues during training.
> > Thank you
> 
> Hi @aditya-malte,
> I have cloned the repo and followed all the steps in preprcoessing.ipnyb and model.ipynb but still got the "loss:nan".

Even I am having the same issue.
